src/storage/v3/conversions.hpp

```diff
@@ -0,0 +1,127 @@
+// Copyright 2022 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include "expr/typed_value.hpp"
+#include "storage/v3/property_value.hpp"
+
+#pragma once
+
+namespace memgraph::storage::v3 {
+
+template <typename TypedValue>
```

Comment:
I genuinly get tricked by this: I missed that `TypedValue` is actually a template type. So I think using a `T` prefix for template type parameters it useful.